documentation/asciidoc/computers/camera/rpicam_apps_building.adoc

```diff
@@ -178,6 +178,23 @@ Finally, run the following command to install your freshly-built `rpicam-apps` b
 $ sudo meson install -C build
 ----
 
+[NOTE]
+======
+rpicam-apps is pre-installed on current 64-bit OS distributions. Before the installation steps above you should remove the pre-installed version.
+
+[source,console]
+----
+$ sudo apt-get remove --purge rpicam-apps
+----
+
+After meson install the ldconfig cache is updated automatically. Running ldconfig without any options will update the cache.
```

Comment:
If it's updated automatically, why does it also need to be run manually?

Reply:
sorry, 
After meson install the ldconfig is updated, but the cache not.